src/components/InvoicesList/InvoicesList.test.js

```diff
@@ -196,4 +197,116 @@ describe('InvoiceList component', () => {
     // Assert
     await waitFor(() => expect(getByText('invoices_list.no_download_msg')).toBeInTheDocument());
   });
+
+  it('should shows a pagination when there are invoices', async () => {
+    // Arrange
+    const invoicesCollection = {
+      items: [
+        {
+          accountId: 'CD0000000073689',
+          product: 'Prod 1',
+          date: new Date('2020-09-25T00:00:00'),
+          currency: 'ARS',
+          amount: 1500,
+          filename: 'invoice_2020-10-05_10.pdf',
+          downloadInvoiceUrl: '',
+        },
+      ],
+      totalItems: 1,
+    };
+
+    const dopplerBillingApiClientDouble = {
+      getInvoices: async () => {
+        return { success: true, value: invoicesCollection };
+      },
+    };
+
+    // Act
+    const { container } = render(
+      <AppServicesProvider
+        forcedServices={{
+          dopplerBillingApiClient: dopplerBillingApiClientDouble,
+        }}
+      >
+        <IntlProvider>
+          <BrowserRouter>
+            <InvoicesList />
+          </BrowserRouter>
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    await waitFor(() => {
+      expect(container.querySelector('.dp-pagination')).toBeInTheDocument();
+    });
+  });
+
+  it('should no shows a pagination when there are no invoices', async () => {
+    // Arrange
+    const invoicesCollection = {
+      items: [],
+      totalItems: 0,
+    };
+
+    const dopplerBillingApiClientDouble = {
+      getInvoices: async () => {
+        return { success: true, value: invoicesCollection };
+      },
+    };
+
+    // Act
+    const { container } = render(
+      <AppServicesProvider
+        forcedServices={{
+          dopplerBillingApiClient: dopplerBillingApiClientDouble,
+        }}
+      >
+        <IntlProvider>
+          <BrowserRouter>
+            <InvoicesList />
+          </BrowserRouter>
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    await waitFor(() => {
+      expect(container.querySelector('.dp-pagination')).not.toBeInTheDocument();
+    });
+  });
+
+  it('should no shows a pagination when the service throw an error', async () => {
```

Comment:
```suggestion
  it('should not show pagination when the service throws an error', async () => {
```

Reply:
I will update the name of the unit tests. Thanks @cbernat

Reply:
Done